spec/shore/client/tokens/access_token_spec.rb

```diff
@@ -0,0 +1,132 @@
+require 'spec_helper'
+
+describe Shore::Client::Tokens::AccessToken do
```

Comment:
I copy-pasted the `spec_helper.rb` from messaging service and fixed all warnings.